megatron/model/transformer.py

```diff
@@ -350,7 +355,7 @@ class FlashSelfAttention(torch.nn.Module):
     def __init__(self, causal=False, softmax_scale=None, attention_dropout=0.0,
                  device=None, dtype=None):
         super().__init__()
-        assert flash_attn_unpadded_func is not None, ('Please install FlashAttention first, '
+        assert flash_attn_unpadded_func is not None or flash_attn_builder is not None, ('Please install FlashAttention first or implement your own fmha'
```

Comment:
I think " or implement your own fmha" should be added after the "e.g., with pip install flash-attn" in the next line? Also what is fmha? Would be helpful to explain a bit more clearly here.